### ISSUE TITLE
issue after aimet exports onnx model

### ISSUE THREAD

**Issue by zhuoran-guo** [quic/aimet#2502]
Hello, I export ONNX models and encoding files using AIMET. While I can successfully export the model, I always encounter a large amount of log information. Do you know the reason behind the generation of this information?
`quant_sim.export(new_dir_path, "aimet_e2e_qat", dummy_input=get_dummy_input_a(model, 128, False), onnx_export_args=onnx_export_args)`
the model can export well but got a lot of log such as :
```
Exported graph: graph(%input0 : Float(1, 3, 128, 128, strides=[49152, 16384, 128, 1], requires_grad=0, device=cpu),
      %input1 : Float(1, 100, strides=[100, 1], requires_grad=0, device=cpu),
      %input2 : Float(1, 100, strides=[100, 1], requires_grad=0, device=cpu),
      %input3 : Float(1, 100, strides=[100, 1], requires_grad=0, device=cpu),
      %input4 : Float(1, 100, strides=[100, 1], requires_grad=0, device=cpu),
      %a : Float(1, 3, 1, 1, strides=[3, 1, 1, 1], requires_grad=0, device=cpu),
      %b : Float(1, 3, 1, 1, strides=[3, 1, 1, 1], requires_grad=0, device=cpu),
      %inner_model.marked_module.encoder.marked_module.model.marked_module.conv_stem.marked_module.weight : Float(32, 3, 3, 3, strides=[27, 9, 3, 1], requires_grad=1, device=cpu),
  ....
  %/module_mul/CustomMarker_output_0 : Float(*, *, *, *, strides=[49152, 16384, 128, 1], requires_grad=0, device=cpu) = aimet_torch::CustomMarker[id="module_mul", leaf="True", start="True", onnx_name="/module_mul/CustomMarker"](%input0), scope: torch.fx.graph_module.GraphModule.__new__.<locals>.GraphModuleImpl::/aimet_torch.onnx_utils.CustomMarker::module_mul # /usr/local/lib/python3.8/dist-packages/aimet_torch/onnx_utils.py:306:0
  %/module_mul/CustomMarker_1_output_0 : Float(*, *, *, *, strides=[3, 1, 1, 1], requires_grad=0, device=cpu) = aimet_torch::CustomMarker[id="module_mul", leaf="True", start="True", onnx_name="/module_mul/CustomMa
```
When I inspect the model's structure, I notice that there are numerous unused (shouldn't exist) inputs, such as: 
`inner_model.marked_module.encoder.marked_module.model.marked_module.conv_stem.marked_module.weight`
<img width="1258" alt="截屏2023-10-16 14 13 57" src="https://github.com/quic/aimet/assets/75294394/e4310759-d0ff-4805-9f1c-78d5fe43681e">

Do you know the reason for this? Will it impact the model's accuracy?

Thank you so much

**Comment by quic-klhsieh** [comment]
Hi @zhuoran-guo , we do see a lot of various messages and warning logs when applying torch.onnx.export which takes place within quantsim.export. Typically these are benign messages and can be ignored.

Which model object are you inspecting at the time of seeing these inputs? As part of quantsim export, we insert new modules within the graph in order to demarcate boundaries between PyTorch modules. However the final exported model, the .onnx file named with the filename you provided in sim.export(), should not contain any such extra modules. Are you seeing these 'marked_module' additions even in this final onnx model?

**Comment by quic-muchhsu** [comment]
Inactive issue. Closing.
